docs/config/dfget_properties.md

```diff
@@ -8,7 +8,7 @@ The following startup parameters are supported for `dfget`
 
 | Parameter | Description |
 | ------------- | ------------- |
-| nodes	| Nodes specify supernodes |
+| nodes	| Nodes specify supernodes with format host:port/weight. Where the host is necessary, the port(default: 8002) and the weight(default:1) are optional. |
```

Comment:
`Nodes specify supernodes with format host:port/weight. Where the host is necessary, the port(default: 8002) and the weight(default:1) are optional.`  ,emmmm I think `Where` will make the gramma wrong ,haha my English is bad,But I feell `Nodes specify supernodes with format host:port/weight. The host is necessary, the port(default: 8002) and the weight(default:1) are optional.` or `Nodes specify supernodes with format host:port/weight where the host is necessary, the port(default: 8002) and the weight(default:1) are optional.` is better.